frontend/src/pages/Login.js

```diff
@@ -0,0 +1,87 @@
+import PageLayout from '../components/PageLayout';
+import styled from 'styled-components/macro';
+import Footer from '../components/Footer';
+import { useState } from 'react';
+import { loginUser } from '../services/loginService';
+import { Redirect } from 'react-router-dom';
+
+export default function Login({ setJwtToken, jwtToken }) {
+    const [username, setUsername] = useState('');
+    const [password, setPassword] = useState('');
+
+    const handleSubmit = (event) => {
+        event.preventDefault();
+        if (!username && !password) {
+            return;
+        }
+        loginUser(username, password).then(setJwtToken);
+        setUsername('');
+        setPassword('');
+    };
+
+    if (jwtToken) {
+        return <Redirect to={'/'} />;
+    }
+
+    return (
+        <PageLayout>
+            <main>
+                <Wrapper>
```

Comment:
I think the main is not needed if you base `Wrapper` on a main tag. And even that may be not needed, you could restyle the PageLayout?

Reply:
I tried to take it out and change my PageLayout class, but it results in wrong rendering on other pages...